bin/run-tests.mjs

```diff
@@ -50,10 +51,10 @@ const browser = await puppeteer.launch({
 console.log('[ci] puppeteer launched');
 
 try {
-  await /** @type {Promise<void>} */ (
-    new Promise(async (fulfill, reject) => {
-      const page = await browser.newPage();
+  const page = await browser.newPage();
```

Comment:
This code is adjusted to run the asynchronous part of the work outside of the `Promise` constructor.
While keeping it inside the `Promise` constructor *works* in this situation, it doesn't behave the
way it looks like it does.